packages/accounts-base/server_main.js

```diff
@@ -7,6 +7,30 @@ import { AccountsServer } from "./accounts_server.js";
 Accounts = new AccountsServer(Meteor.server, { ...Meteor.settings.packages?.accounts, ...Meteor.settings.packages?.['accounts-base'] });
 // TODO[FIBERS]: I need TLA
 Accounts.init().then();
+
+/**
+ * @summary Find a user by one of their email addresses.
+ * @locus Server
+ * @param {String} email The email address to look for
+ * @param {Object} [options]
+ * @param {Object} options.fields Limit the fields to return from the user document
+ * @returns {Promise<Object>} A user if found, else null
+ * @memberof Accounts
+ * @importFromPackage accounts-base
+ */
+Accounts.findUserByEmail = Accounts.findUserByEmail;
```

Comment:
I get the JSDocs addition, but I don't understand the code part. Wouldn't it be better to add the docs on AccountsServer or do `Accounts.findUserByEmail = AccountsServer.findUserByEmail;`?